source/docs/articles/organizations/base-domains.md

```diff
@@ -26,6 +26,14 @@ Sites associated with your organization will receive the custom base URL for all
 
 **Environment URLs are permanent.** If an organization is removed as the supporting organization, any environment created during its association will keep the original URL after removal. Paid sites can add custom domains to any environment, as a workaround for those wishing to use different URL's after launch and disassociation of the site with the organization.
 
+## Security Considerations
+
+If you run sites on the subdomains as your primary website, you should be aware of some security considerations:
+
+* The sites on the subdomains may be able to read cookies set on your marketing site
```

Comment:
I think the language should be "primary site" rather than "marketing site" to clarify that it's the top level domain, not a subdomain.